README.md

```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
     ```
     Interfacing Options -> SPI
    ```
-2. Install eInk display drivers
+2. Installing dependencies via apt
     ```
-    git clone https://github.com/waveshare/e-Paper.git
-    cd e-Paper-master/RaspberryPi_JetsonNano/python/
-    pip3 install -e $(pwd)
+    sudo apt update
+    sudo apt install -y libtiff5 libopenjp2-7 libopenjp2-7 libatlas-base-dev
```

Comment:
It looks like, these libraries are redundant.
All dependencies required for the drivers to work are described here:
https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT

Which could be boiled down to the following: 
```
sudo apt-get install python3-pip python3-pil python3-numpy
pip3 install RPi.GPIO spidev
```
The script doesn't require any additional libraries apart from listed above.

Reply:
Sounds great!
Would still be a great improvement, since new users wouldn't need to do the research by themselves.

I realized, that I've solved the problems are more complicated way.